src/reference.yaml

```diff
@@ -5397,6 +5397,12 @@ paths:
               category: date_of_birth
               message: date_of_birth should be of date type
               params: '{ "type" => "date" }'
+            - code: date_of_birth_is_required
+              category: date_of_birth
+              message: >-
+                Date of Birth is a required field where account type is individual. Please provide date of birth
```

Comment:
@maria-thom the message here needs to match the response from the API call. Has the response also been updated to have this message? I do not see the change